.github/workflows/reusable-unit.yml

```diff
@@ -75,9 +76,13 @@ jobs:
         run: yarn ui build
 
       - name: Build react-core package
-        if: ${{ matrix.package == 'react' || matrix.package == 'react-native' }}
+        if: ${{ matrix.package == 'react' || matrix.package == 'react-native' || matrix.package == 'react-storage' }}
         run: yarn react-core build
 
+      - name: Build react package
+        if: ${{ matrix.package == 'react-storage' && matrix.package != 'react'}}
```

Comment:
```suggestion
        if: ${{ matrix.package == 'react-storage' }}
```
This seems suffice

Reply:
nit: could also add a comment to help people understand

I know it probably because `react-storage` depends on `react`.